mcs/ilasm/scanner/ILTables.cs

```diff
@@ -321,6 +321,7 @@ public static Hashtable Keywords
                                 keywords ["legacy"] = new ILToken (Token.K_LEGACY, "legacy");
                                 keywords ["library"] = new ILToken (Token.K_LIBRARY, "library");
                                 keywords ["auto"] = new ILToken (Token.K_AUTO, "auto");
+                                keywords ["aggressiveinlining"] = new ILToken (Token.K_AGGRESSIVEINLINING, "aggressiveinlining");

```

Comment:
is this keyword documented somewhere? I looked at ECMA-335 and couldn't find it (`noinlining` and `nooptimization` are there in `II.15.4.3`)

Reply:
The last ECMA-335 update was years ago and it's not tracking anything from .net 4 or newer

Reply:
yep, I just wanted to make sure you didn't invent this keyword right now 😄 

This looks like a good reference: https://github.com/dotnet/coreclr/blob/b13b7c16a2c3cea1557f941000b9c054a7664f92/src/inc/il_kywd.h#L97